lib/Scvmm/miq_scvmm_parse_powershell.rb

```diff
@@ -0,0 +1,11 @@
+
+class MiqScvmmParsePowershell
```

Comment:
I agree.  One of the things I want to do is move her class under lib instead of vmdb where it lives today.  it is my understanding that code under lib shouldn't be calling out to code under vmdb, although the reverse is the norm.
(One of the many "In Progress" items making this a WIP PR).